README.md

```diff
@@ -4,6 +4,34 @@
 ## Source Control Infrastructure
 For this project, we will be Git for source control with a feature branch style of source control. 
 
+### Step-To-Step Git Instructions to Make Changes
+1. Supposed you have already had your git setup properly
+2. Make sure your local repo is updated with master.
+
+    ` git pull origin master`
+3. Create and move to a **new branch**
+
+    ` git branch new-sample-branch`
+
+    ` git checkout new-sample-branch`
+4. Now that you are on your own feature branch, make whatever changes you want!
+5. Once you complete your feature, **add, commit and push** your change to Github. If you do everything correctly, you would see your change in Github branch: features/new-sample-branch
+    
+    `git add .`
+
+    `git commit -a`
```

Comment:
dangerous to do, use ```git commit -m "your update message here"```

Reply:
I wonder why it is dangerous to do -a here. If you have added all why not just commit them

Reply:
I could be wrong but does this default to no commit message? If there is no commit message it would make tracing back commits difficult

Reply:
this would open up your text editor and you could write your commit message there. I think the  -a flag just means committing all staged changes. But really `-m` would work nearly the same way without the editor. Maybe `git commit -am "your commit msg"` would be best here